demo/cache/cache-controller.js

```diff
@@ -9,9 +10,15 @@ app.controller('CacheCtrl', function($scope, $rootScope, $http, File, FDP, HttpE
     File.read('data/query/listCachedFdps.sparql').then(function(query) {
       HttpEndpoint.query(query).then(function(response) {
         response.data.results.bindings.forEach(function(binding) {
+        	var fdpFromCookies = $cookies.getObject(btoa(binding.fdp.value));
```

Comment:
Do you need the `btoa` in there? I believe the cookies module has ways to serialize and deserialize objects built in.

Reply:
But when I stored the fdp object to the cookie I used btoa(FDPs url) as a key. I can't just use url as a key, I noticed that cookies module encoding the url